From c82731710ca202ea68657338636d06e853224732 Mon Sep 17 00:00:00 2001
From: Elaine Zhang <zhangqing@rock-chips.com>
Date: Thu, 5 Nov 2020 18:05:22 +0800
Subject: [PATCH] arm64: dts: rockchip: rk3399pro-evb: remove
 regulator-xxx-microvolt for SWITCH pmic node

Switch is just a function of switching. There is no voltage setting
function. Voltage getting is the supply voltage.

Fixes: 3fc99e38fdbf ("regulator: resolve supply after creating
regulator")

Signed-off-by: Elaine Zhang <zhangqing@rock-chips.com>
Change-Id: I83f487908931f40bbeb4323ad671d3a446247377
---
 .../boot/dts/rockchip/rk3399pro-evb-lp4-v11-linux.dts      | 7 -------
 arch/arm64/boot/dts/rockchip/rk3399pro-evb-v10-linux.dts   | 7 -------
 arch/arm64/boot/dts/rockchip/rk3399pro-evb-v10.dts         | 7 -------
 arch/arm64/boot/dts/rockchip/rk3399pro-evb-v11-linux.dts   | 7 -------
 arch/arm64/boot/dts/rockchip/rk3399pro-evb-v11.dts         | 7 -------
 5 files changed, 35 deletions(-)

diff --git a/arch/arm64/boot/dts/rockchip/rk3399pro-evb-lp4-v11-linux.dts b/arch/arm64/boot/dts/rockchip/rk3399pro-evb-lp4-v11-linux.dts
index feaa4909e535..db77fb3c4072 100644
--- a/arch/arm64/boot/dts/rockchip/rk3399pro-evb-lp4-v11-linux.dts
+++ b/arch/arm64/boot/dts/rockchip/rk3399pro-evb-lp4-v11-linux.dts
@@ -674,22 +674,15 @@
 			};
 
 			vcc5v0_usb: SWITCH_REG1 {
-				regulator-min-microvolt = <5000000>;
-				regulator-max-microvolt = <5000000>;
-
 				regulator-name = "vcc5v0_usb";
 				regulator-state-mem {
 					regulator-on-in-suspend;
-					regulator-suspend-microvolt = <5000000>;
 				};
 			};
 
 			vccio_3v3: SWITCH_REG2 {
 				regulator-always-on;
 				regulator-boot-on;
-				regulator-min-microvolt = <3300000>;
-				regulator-max-microvolt = <3300000>;
-
 				regulator-name = "vccio_3v3";
 				regulator-state-mem {
 					regulator-off-in-suspend;
diff --git a/arch/arm64/boot/dts/rockchip/rk3399pro-evb-v10-linux.dts b/arch/arm64/boot/dts/rockchip/rk3399pro-evb-v10-linux.dts
index c43a84428136..1b4301261c3c 100644
--- a/arch/arm64/boot/dts/rockchip/rk3399pro-evb-v10-linux.dts
+++ b/arch/arm64/boot/dts/rockchip/rk3399pro-evb-v10-linux.dts
@@ -575,22 +575,15 @@
 			};
 
 			vcc5v0_usb: SWITCH_REG1 {
-				regulator-min-microvolt = <5000000>;
-				regulator-max-microvolt = <5000000>;
-
 				regulator-name = "vcc5v0_usb";
 				regulator-state-mem {
 					regulator-on-in-suspend;
-					regulator-suspend-microvolt = <5000000>;
 				};
 			};
 
 			vccio_3v3: SWITCH_REG2 {
 				regulator-always-on;
 				regulator-boot-on;
-				regulator-min-microvolt = <3300000>;
-				regulator-max-microvolt = <3300000>;
-
 				regulator-name = "vccio_3v3";
 				regulator-state-mem {
 					regulator-off-in-suspend;
diff --git a/arch/arm64/boot/dts/rockchip/rk3399pro-evb-v10.dts b/arch/arm64/boot/dts/rockchip/rk3399pro-evb-v10.dts
index 0306547ea062..6585e48c0b76 100644
--- a/arch/arm64/boot/dts/rockchip/rk3399pro-evb-v10.dts
+++ b/arch/arm64/boot/dts/rockchip/rk3399pro-evb-v10.dts
@@ -604,22 +604,15 @@
 			vcc5v0_usb: SWITCH_REG1 {
 				regulator-always-on;
 				regulator-boot-on;
-				regulator-min-microvolt = <5000000>;
-				regulator-max-microvolt = <5000000>;
-
 				regulator-name = "vcc5v0_usb";
 				regulator-state-mem {
 					regulator-on-in-suspend;
-					regulator-suspend-microvolt = <5000000>;
 				};
 			};
 
 			vccio_3v3: SWITCH_REG2 {
 				regulator-always-on;
 				regulator-boot-on;
-				regulator-min-microvolt = <3300000>;
-				regulator-max-microvolt = <3300000>;
-
 				regulator-name = "vccio_3v3";
 				regulator-state-mem {
 					regulator-off-in-suspend;
diff --git a/arch/arm64/boot/dts/rockchip/rk3399pro-evb-v11-linux.dts b/arch/arm64/boot/dts/rockchip/rk3399pro-evb-v11-linux.dts
index 7808dafe0c43..615f0c998c16 100644
--- a/arch/arm64/boot/dts/rockchip/rk3399pro-evb-v11-linux.dts
+++ b/arch/arm64/boot/dts/rockchip/rk3399pro-evb-v11-linux.dts
@@ -585,22 +585,15 @@
 			};
 
 			vcc5v0_usb: SWITCH_REG1 {
-				regulator-min-microvolt = <5000000>;
-				regulator-max-microvolt = <5000000>;
-
 				regulator-name = "vcc5v0_usb";
 				regulator-state-mem {
 					regulator-on-in-suspend;
-					regulator-suspend-microvolt = <5000000>;
 				};
 			};
 
 			vccio_3v3: SWITCH_REG2 {
 				regulator-always-on;
 				regulator-boot-on;
-				regulator-min-microvolt = <3300000>;
-				regulator-max-microvolt = <3300000>;
-
 				regulator-name = "vccio_3v3";
 				regulator-state-mem {
 					regulator-off-in-suspend;
diff --git a/arch/arm64/boot/dts/rockchip/rk3399pro-evb-v11.dts b/arch/arm64/boot/dts/rockchip/rk3399pro-evb-v11.dts
index 84b79d55869a..c3e9cc29fc62 100644
--- a/arch/arm64/boot/dts/rockchip/rk3399pro-evb-v11.dts
+++ b/arch/arm64/boot/dts/rockchip/rk3399pro-evb-v11.dts
@@ -589,22 +589,15 @@
 			vcc5v0_usb: SWITCH_REG1 {
 				regulator-always-on;
 				regulator-boot-on;
-				regulator-min-microvolt = <5000000>;
-				regulator-max-microvolt = <5000000>;
-
 				regulator-name = "vcc5v0_usb";
 				regulator-state-mem {
 					regulator-on-in-suspend;
-					regulator-suspend-microvolt = <5000000>;
 				};
 			};
 
 			vccio_3v3: SWITCH_REG2 {
 				regulator-always-on;
 				regulator-boot-on;
-				regulator-min-microvolt = <3300000>;
-				regulator-max-microvolt = <3300000>;
-
 				regulator-name = "vccio_3v3";
 				regulator-state-mem {
 					regulator-off-in-suspend;
-- 
2.35.3

